contracts/MerkleTree.sol

```diff
@@ -41,5 +41,7 @@ contract MerkleTree {
      * @notice Verify that a specific leaf element is part of the Merkle Tree at a specific position in the tree
      * @return a boolean value representing the success or failure of the operation
      */
-    function verifyAtPosition()
+    // function verifyAtPosition() {
+    //     // TODO
+    // }
```

Comment:
This is missing and should be added, just commenting it out here to make the tests run.